joss/paper.md

```diff
@@ -28,59 +28,228 @@ bibliography: paper.bib
+Sometimes the dynamics are instead posed as a set of differential algebraic equations, where some residual equations need to be satisfied implicitly in order to solve ODE.
+One application of this approach is the method of differential inclusions, in which the state time-history is posed as a dynamic control, and the traditional control variables needed to achieve that trajectory are found using a nonlinear solver within the ODE [@Seywald1994].
 Support for implicit calculations gives users more freedom to pose dynamics in more natural ways, but typically causes numerical and computational cost challenges in an optimization context, especially when finite-differences are used to compute derivatives for the optimizer.
-Dymos employs analytic derivatives to overcome these challenges and makes such methods computationally feasible.
-To achieve this, dymos makes extensive use of OpenMDAO's built in analytic derivative features, its non-linear solver library, and its support for gradient-based optimization.
+Some optimal control libraries tackle this numerical challenge with a monolithic algorithmic differentiation[@griewank2003mathematical] approach.
+While effective, the monolithic nature is both less efficient[@mader2008adjoint; @kenway2019effective] and less flexible.
+
+Instead, Dymos uses a modular approach that allows users to select any combination of finite-difference, complex-step [@Martins2003CS], algorithmic differentiation, and hand differentiation that suits their modeling needs.
+This flexibility is achieved by leveraging the data passing and efficient differentiation fearures of NASA's OpenMDAO framework[@Gray2019a]. 
+Dymos extends OpenMDAO by adding differentiated time-integration schemes along with the necessary APIs for users to plug their ODEs and DAEs into those schemes.
+It also provides further APIs to enable in the input and output of information from the optimal control problem, in order to couple with the physical design models. 
 
-Dymos can be used stand-alone, or as a building block in a larger model where a significant portion of the optimization is focused on some non-controls aspect with a trajectory added to enforce some constraint upon the design.
-In some contexts, you may hear this kind of problem referred to as co-design, controls-co-design, or multidisciplinary design optimization.
 
 ## The dymos perspective on optimal control
 
-In dymos trajectories are subdivided into "phases," and within each phase, boundary or path constraints may be applied to any output of the system dynamics.
-In addition to typical collocated dynamic control variables, dymos offers the ability to use variable order polynomial controls that are useful in forcing a lower-order control solution upon a phase of the trajectory.
+Dymos breaks the trajectory of a system is into chunks of time called _phases_.
+Breaking the trajectory into phases provides several capabilities.
```

Comment:
"provides several capabilities"

I think it provides advantages, or maybe makes it easier to implement certain capabilities, but it doesn't provide capabilities.